features/user_views_artists.feature

```diff
@@ -5,3 +5,10 @@ Feature: User views artists listing
     When I open the app
     And I want to see a listing of the artists
     Then I should see 12 or fewer artists
+
+  Scenario: A user views an artist
+    Given the app has a few artists
+    When I open the app
```

Comment:
Since this step and the one before it are both being used by all the scenarios in the test, they can be moved into a `Background` group [like this](https://github.com/BaltimorePublicArtCommons/baltimore_public_art_commons/blob/master/features/user_searches_for_items.feature#L3-L7).